.github/workflows/publish-release-artifact.yml

```diff
@@ -58,7 +58,7 @@ jobs:
           wget -q https://get.jenkins.io/${REPO}/${PROJECT_VERSION}/${FILE_NAME}
       - name: Upload Release Asset
         id: upload-war
-        uses: actions/upload-release-asset@v1.0.2
+        uses: actions/upload-release-asset@v1
```

Comment:
Is there are particular reason why core uses such an ancient version? The repository providing it is EOL since early 2021, therefore pinning this specific version has bar no effect on it.